README.md

```diff
@@ -181,19 +185,18 @@ jobs:
           fermyon_token: ${{ secrets.FERMYON_CLOUD_TOKEN }}
 ```
 
-
-## `fermyon/action/spin/preview`
+## Deploy preview of Spin app to Fermyon Cloud - `fermyon/action/spin/preview@v1`
 
 Build and deploy the `spin` app preview to Fermyon Cloud
 
 ### Inputs
 
-| Name          | Required | Description                                                                       | Default   |
-| ------------- | -------- | --------------------------------------------------------------------------------- | --------- |
-| fermyon_token | Required | Fermyon Cloud Personal access token for deploying the `spin` app to Fermyon Cloud | -         |
-| manifest_file | Optional | Path to `spin.toml`. Used with the `build`/`deploy` command.                      | spin.toml |
-| github_token  | Required | The `GitHub` token for updating comment on PR                                     | -         |
-| undeploy      | Optional | If true, removes the preview deployment from Fermyon Cloud                        | -         |
+| Name          | Required | Default   | Description                                                                       |
+| ------------- | -------- |--------- | --------------------------------------------------------------------------------- |
+| fermyon_token | True | -         | Fermyon Cloud Personal access token for deploying the `spin` app to Fermyon Cloud |
+| manifest_file | False | spin.toml | Path to `spin.toml`. Used with the `build`/`deploy` command.                      |
```

Comment:
In other actions we don't explain that `spin.toml` is used with the build/deploy command.  Should we be consistent?  (I'd say drop it here.)